Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6-slim
```

Comment:
```suggestion
FROM python:3.8-slim
```

Reply:
🦕

Reply:
i chose 3.6 here because that is the minimum supported version in `setup.py` and i thought this would be best for ensuring compatibility.